src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -163,7 +163,7 @@ public EditPersonDescriptor(EditPersonDescriptor toCopy) {
          * Returns true if at least one field is edited.
          */
         public boolean isAnyFieldEdited() {
-            return CollectionUtil.isAnyNonNull(name, phone, email, address, tags, comments);
+            return CollectionUtil.isAnyNonNull(name, phone, email, telegramHandle, tags);
```

Comment:
Why is comments being removed? This would introduce a regression, as an edit command that only changes comments will not be recognised, which does not sound right.

Reply:
apologies for the merging regression